src/fi_tostr.c

```diff
@@ -72,7 +72,7 @@
 
 static void ofi_tostr_fid(const char *label, char *buf, const struct fid *fid)
 {
-	if (!fid || !FI_CHECK_OP(fid->ops, struct fi_ops, tostr))
+	if (!fid || !fid->ops || !FI_CHECK_OP(fid->ops, struct fi_ops, tostr))
```

Comment:
I wonder if we should move this check into FI_CHECK_OPS() instead.

Reply:
Yep - I think that is probably a better fix. Changing that now...